src/Middleware/OutputCaching/src/Memory/MemoryOutputCacheStore.cs

```diff
@@ -81,14 +90,47 @@ public ValueTask SetAsync(string key, byte[] value, string[]? tags, TimeSpan val
 
         void SetEntry()
```

Comment:
nit: make this local function static and pass the state in

Reply:
Was not able to make it static. Made it an instance one though. It needs `_cache` and `options`.

Reply:
That works, but not sure why static wouldn't work, you just pass in `_cache` and `options`.

Reply:
Sure, everything can be static if you pass the state. Why do you prefer it here?

Reply:
I don't in this case, just making sure you know since you said
> Was not able to make it static